tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Documents/DocumentsApiSteps.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
Is this using statement used? The reason I ask is the changes here are nicer formatting & nothing new brought in.

Reply:
No, it's not. Unusually for me I missed that one.

Reply:
I must have made some changes to this class for debugging or testing purposes at some point and missed undoing them. I'll just remove the unnecessary using.

Reply:
Yeah I figured as much 👍